### PR TITLE
NewAccountDialog + Custom Currency Improvements

### LINKED ISSUE #573: New Account Dialog Improvements
- Offer the option to import data from a file
- Overwrite files switch ?

### PR DESCRIPTION
**Pull request by nlogozzo**
Closes #573 
+
Cleaned up UI for custom currency

### CODE REVIEW

File: NickvisionMoney.GNOME/Views/AccountSettingsDialog.cs [hunk 1]
```diff
@@ -125,34 +120,44 @@ private AccountSettingsDialog(Gtk.Builder builder, AccountSettingsDialogControll
...
+                }
                 if (!_constructing)
                 {
                     Validate();
                 }
             }
         };
-        _customDecimalSeparatorDropDown.SetModel(Gtk.StringList.New(new string[3] { ".", ",", _p("DecimalSeparator", "Other") }));
-        _customDecimalSeparatorDropDown.OnNotify += (sender, e) =>
+        _customDecimalSeparatorRow.SetModel(Gtk.StringList.New(new string[3] { ". ", ", ", _p("DecimalSeparator", "Other") }));
```
Comment (DaPigGuy) [inline NickvisionMoney.GNOME/Views/AccountSettingsDialog.cs:153]: Any reason this isn't defined in the Blueprint?

File: NickvisionMoney.GNOME/Views/AccountSettingsDialog.cs [hunk 2]
```diff
@@ -175,14 +180,14 @@ private AccountSettingsDialog(Gtk.Builder builder, AccountSettingsDialogControll
                 }
             }
         };
-        _customGroupSeparatorDropDown.SetModel(Gtk.StringList.New(new string[5] { ".", ",", "'", _p("GroupSeparator", "None"), _p("GroupSeparator", "Other") }));
-        _customGroupSeparatorDropDown.OnNotify += (sender, e) =>
+        _customGroupSeparatorRow.SetModel(Gtk.StringList.New(new string[5] { ". ", ", ", "' ", _p("GroupSeparator", "None"), _p("GroupSeparator", "Other") }));
```
Comment (DaPigGuy) [inline NickvisionMoney.GNOME/Views/AccountSettingsDialog.cs:183]: Same here

**Review by DaPigGuy — CHANGES_REQUESTED** [review]

File: NickvisionMoney.GNOME/Blueprints/new_account_dialog.blp [hunk 3]
```diff
@@ -102,14 +102,24 @@ Adw.Window _root {
                     styles ["flat"]
                   }
                 }
+
+                Adw.ActionRow _overwriteRow {
+                  title: _("Overwrite existing accounts");
```
[the remark this answers is not in the document]
Comment (nlogozzo) [reply NickvisionMoney.GNOME/Blueprints/new_account_dialog.blp:107]: Yes, I'll fix that.

File: NickvisionMoney.GNOME/Views/AccountSettingsDialog.cs (hunk 1 above)
Comment (nlogozzo) [reply NickvisionMoney.GNOME/Views/AccountSettingsDialog.cs:153]: No, no reason. I'll move it to blueprint.

**Pull request merged**